### PR TITLE
Add/more descriptions to notebooks

### PR DESCRIPTION
**Pull request by dthiex**
Adding some more text to the single steps of the notebook to make it easier to understand what's being shown.

### CODE REVIEW

File: notebooks/NDVI example.ipynb [hunk 1]
```diff
@@ -18,6 +27,14 @@
     "import jsbeautifier"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "id": "e534ff79",
+   "metadata": {},
+   "source": [
+    "Connect to the openEO backened (2) to create a datacube (3)."
```
Comment (Ardweaden) [inline notebooks/NDVI example.ipynb]: Typo: backend.

File: notebooks/NDVI example.ipynb [hunk 2]
```diff
@@ -59,6 +84,14 @@
     "datacube_ndvi = datacube.reduce_dimension(reducer = ndvi_function, dimension = \"bands\")"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "id": "4303f311",
+   "metadata": {},
+   "source": [
+    "Converte process graph into json format and load into variable."
```
Comment (Ardweaden) [inline notebooks/NDVI example.ipynb]: Typo: convert.

File: notebooks/NDVI example.ipynb [hunk 3]
```diff
@@ -69,6 +102,41 @@
...
+    "subgraph_evalscripts = convert_from_process_graph(process_graph_ndvi)\n",
+    "print(subgraph_evalscripts)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "196c3db0",
+   "metadata": {},
+   "source": [
+    "Save evalscript as string and print redable version."
```
Comment (Ardweaden) [inline notebooks/NDVI example.ipynb]: Typo: readable

I wouldn't say we "save evalscript as string", we generate the evalscript string.

File: notebooks/Basic example.ipynb [hunk 4]
```diff
@@ -23,7 +41,15 @@
...
-    "    return pg\n"
+    "    return pg"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "607eb405",
+   "metadata": {},
+   "source": [
+    "Load process graph, convert to evalscript and print generated evalscript."
```
Comment (Ardweaden) [inline notebooks/Basic example.ipynb]: We could add the `jsbeautifier` here too, to fix up the evalscript formatting a bit.

File: notebooks/openeo_vito_example.ipynb [hunk 5]
```diff
@@ -17,9 +26,17 @@
     "import matplotlib.pyplot as plt"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "id": "0c2bfc66",
+   "metadata": {},
+   "source": [
+    "Connect to the openEO backened (2) to create a test polygon (3) and a datacube (4)."
```
Comment (Ardweaden) [inline notebooks/openeo_vito_example.ipynb]: Typo: backend

We don't really need this to create the test polygon.

File: notebooks/openeo_vito_example.ipynb [hunk 6]
```diff
@@ -75,17 +92,25 @@
     "process_graph = json.loads(sen2cor_masked.to_json())[\"process_graph\"]"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "id": "4689a33e",
+   "metadata": {},
+   "source": [
+    "Convert process graph into an evalscript and print info about the created evalscript. The 'invalid_node_id' tells us up to which node the process graph could be converted. If it's \"None\", the whole process graph could be converted into an evalscript."
```
Comment (Ardweaden) [inline notebooks/openeo_vito_example.ipynb]: Again we could use the `jsbeautifier` here, as we do in the NDVI evalscript.

**Review by Ardweaden — CHANGES_REQUESTED** [review]

File: notebooks/NDVI example.ipynb (hunk 1 above)
Comment (dthiex) [reply notebooks/NDVI example.ipynb]: Fixed

File: notebooks/NDVI example.ipynb (hunk 2 above)
Comment (dthiex) [reply notebooks/NDVI example.ipynb]: Fixed

File: notebooks/Basic example.ipynb (hunk 4 above)
Comment (dthiex) [reply notebooks/Basic example.ipynb]: Agree, added

File: notebooks/NDVI example.ipynb (hunk 3 above)
Comment (dthiex) [reply notebooks/NDVI example.ipynb]: Fixed (both)

File: notebooks/openeo_vito_example.ipynb (hunk 5 above)
Comment (dthiex) [reply notebooks/openeo_vito_example.ipynb]: Fixed (and polygon part rephrased)

File: notebooks/openeo_vito_example.ipynb (hunk 6 above)
Comment (dthiex) [reply notebooks/openeo_vito_example.ipynb]: Added

File: notebooks/openeo_vito_example.ipynb (hunk 5 above)
Comment (Ardweaden) [reply notebooks/openeo_vito_example.ipynb]: It's "backned" now 😄 

fftm after fixing

**Review by Ardweaden — DISMISSED** [review]
I need to leave a comment to request changes.

But feel free to merge after fixing the typo.

File: notebooks/openeo_vito_example.ipynb (hunk 5 above)
Comment (dthiex) [reply notebooks/openeo_vito_example.ipynb]: 🤦‍♂️

**Pull request merged**